wordpress-mobile/WordPress-Android
# Update stats tab offscreenPageLimit to fix datepicker problem

### Problem
The screen was stuck at the loading state on the Views & Visitors, Total Likes detail screens. The reason could be this,`selectedDate` is null here on these screens:
https://github.com/wordpress-mobile/WordPress-Android/blob/898f0354acb7bbb2a25438e1f95c40c4492d4c25/WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/granular/GranularStatefulUseCase.kt#L31-L34
When selecting other tabs on the Insights screen, you cause WEEKS tabs caching. Then Views & Visitors works since Views & Visitors detail uses weeks granularity, and it can take advantage of WEEKS tabs caching on data selector.

### Solution
I increased stats tab `offscreenPageLimit` parameter. Now, when you open stats screen, WEEKS tab is being loaded and detail screens work well.
This is a quick solution: we can call it a workaround. But it doesn't harm anything. In addition, the WEEKS tab will be in cache now, and opening faster at the first launch. 

To test:
1. Kill the app if it's opened and relaunch.
2. Go Stats
3. Insights tab must be selected (If not, select Insights, kill the app, relaunch, Insights will be opened next time)
4. Click VIEW MORE in Views & Visitors
5. Ensure the loading state isn't stuck and cards appear.

## Regression Notes
1. Potential unintended areas of impact
Redundant calls. It causes an extra call for a hidden tab, but not very important. That call can also cause faster loading when that tab is selected. 

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Tested manually. 

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
